README.md

```diff
@@ -32,27 +34,27 @@ Core Server (i.e. SERVER) project are **public**.
 ## Installation
 `MongoSwift` works with Swift 4.0+.
 
-### FIRST: Install the MongoDB C Driver
-Because the driver wraps the MongoDB C driver, using it requires having the C driver's two components, `libbson` and `libmongoc`, installed on your system. The minimum required version of the C Driver is **1.13.0**.
+### OS X and Linux
+
+Installation on OS X and Linux is supported via [Swift Package Manager](https://swift.org/package-manager/).
+
+#### Step 1: Install the MongoDB C Driver
+The driver wraps the MongoDB C driver, and using it requires having the C driver's two components, `libbson` and `libmongoc`, installed on your system. The minimum required version of the C Driver is **1.13.0**.
 
 On a Mac, you can install both components at once using [Homebrew](https://brew.sh/):
 `brew install mongo-c-driver`.
 
-Or on Linux, use `apt-get` to install `libmongoc` (which includes `libbson` as a dependency) and `pkg-config` (which enables Swift Package Manager to find the components):
+On Linux, use `apt-get` to install `libmongoc` (which includes `libbson` as a dependency) and `pkg-config` (which enables Swift Package Manager to find the components):
```

Comment:
This isn't new to this PR, but I don't think Debian or Ubuntu (assuming `apt-get`) are going to provide the necessary version of libmongoc. It may be easier to just point Linux users to http://mongoc.org/libmongoc/current/installing.html and skip the `apt-get install libmongoc-1.0.0` suggestion entirely. 

When we ultimately get libmongoc builds in MongoDB's own [package repositories](https://repo.mongodb.org), I expect that page will also be updated with instructions accordingly.

Reply:
good point - this is something we have encountered a few times already in issues on the repo. however, the instructions there also point users to `apt-get`. I've updated this say that `apt-get`'s versions are currently insufficient and point users to the instructions for build/install from source instead.

Reply:
Using looser language like "If `apt-get`'s versions are too old, follow manual build instructions" may be better. Then this text can remain accurate even if `apt-get` is updated down the line.

Reply:
Ah, just saw https://github.com/mongodb/mongo-swift-driver/pull/147#discussion_r. Simply referring to libmongoc's instructions seems best.

Reply:
ok, I've made this vaguer.